### PR TITLE
Docs: Update Storybook docs to differentiate between the Webpack and Vite versions

### PR DESCRIPTION
**Pull request by arimendelow**
This PR updates the Storybook docs to differentiate between the Webpack and Vite versions.

### CODE REVIEW

File: docs/docs/storybook/vite.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- `web/.storybook/preview-body.html`
+  - This is required to change the `id` of the root div to `redwood-app`, which is what the entry file used by Vite requires.
+- `web/src/auto-imports.d.ts`
+  - Created by [`unplugin-auto-import`](https://github.com/unplugin/unplugin-auto-import), which is used by the Framework Package [here](https://github.com/redwoodjs/redwood/blob/main/packages/storybook/src/plugins/auto-imports.ts).
+
+Once Storybook is all set up, it'll spin up on port `7910`.
+
+## Migrating from Storybook Webpack to Storybook Vite
+
+If you don't have any of your own [Storybook configuration](https://redwoodjs.com/docs/storybook#configuring-storybook), you should be good to go. The Out of Box experience should be the same, and please [let us know](https://github.com/redwoodjs/redwood/issues/new?assignees=&labels=bug%2Fneeds-info&projects=&template=bug-report.yml&title=%5BBug%3F%5D%3A+) if you run into any issues.
```
Comment (dac09) [inline docs/docs/storybook/vite.md:35]:
```suggestion
If you don't have any of your own [Storybook configuration](https://redwoodjs.com/docs/storybook#configuring-storybook), you should be good to go - no changes needed. The Out of Box experience should be the same, and please [let us know](https://github.com/redwoodjs/redwood/issues/new?assignees=&labels=bug%2Fneeds-info&projects=&template=bug-report.yml&title=%5BBug%3F%5D%3A+) if you run into any issues.
```

File: docs/docs/storybook/webpack.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
-Storybook enables a kind of frontend-first, component-driven development workflow that we've always wanted.
-By developing your UI components in isolation, you get to focus exclusively on your UI's needs,
-saving you from getting too caught up in the details of your API too early.
+These docs are for the original Storybook integration, which uses Webpack and stores its config files in the framework.
 
-Storybook also makes debugging a lot easier.
-You don't have to start the dev server, login as a user, tab through dropdowns, and click buttons just for that one bug to show up.
-Or render a whole page and make six GraphQL calls just to change the color of a modal.
-You can set it all up as a story, tweak it there as you see fit, and even test it for good measure.
+If you're still using this, please consider upgrading to the our latest Storybook integration, which uses Vite. This comes with two primary advantages:
```
Comment (dac09) [inline docs/docs/storybook/webpack.md:9]:
```suggestion
> If you're still using this, please consider upgrading to the our latest Storybook integration, which uses Vite. This comes with two primary advantages:
> - Storybook is now much faster: [Storybook Performance: Vite vs Webpack 4](https://storybook.js.org/blog/storybook-performance-from-webpack-to-vite/)
> - You get full ownership over your Storybook config!
```

Comment (dac09) [reply docs/docs/storybook/webpack.md:9]: This 'll just highlight it :) I think I got the syntax right!

**Review by dac09 — APPROVED** [review]

**Comment by dac09** [comment]
Left a couple of minor optional suggestions @arimendelow - will let you merge.

File: docs/docs/storybook/webpack.md (hunk 2 above)
Comment (arimendelow) [reply docs/docs/storybook/webpack.md]: ahhh yup perfect :)

Screenshot 2024-07-01 at 10 41 58

**Pull request merged**